packages/core/src/components/table/table-body-row-expandable/table-body-row-expandable.tsx

```diff
@@ -73,7 +97,35 @@ export class TdsTableBodyRowExpandable {
     }
   }
 
+  @Watch('expanded')
+  watchExpanded(newValue: boolean) {
+    console.log('watch');
+    if (newValue !== this.isExpanded) {
+      this.isExpanded = newValue;
+      this.tdsChange.emit({ rowId: this.rowId, isExpanded: this.isExpanded });
+    }
+  }
```

Comment:
```suggestion
  @Watch('expanded')
  watchExpanded(newValue: boolean) {  
    if (newValue !== this.isExpanded) {
      this.isExpanded = newValue;
      this.tdsChange.emit({ rowId: this.rowId, isExpanded: this.isExpanded });
    }
  }
```
Just removing console.log...